ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/DataStorageConfiguration.java

```diff
@@ -31,6 +31,12 @@ public interface DataStorageConfiguration {
           .unstable(Unstable.DEFAULT)
```

Comment:
why forest will have max layer to load ?

Reply:
Found out why it's needed when the build broke after removing it. It's needed because bonsaiMaxLayersToLoad is a required attribute for the Immutable value. We could make it optional and add some validation to ensure it's specified when using Bonsai mode. A better approach is probably to move all the Bonsai configuration into an Optional<BonsaiDataStorageConfiguration> as part of DataStorageConfiguration.

Not sure it's worth the effort to make a change. What do you think?

Reply:
I think it's fine as it was like that before